x509/build/build.rs

```diff
@@ -64,19 +64,19 @@ fn gen_fmc_alias_cert(out_dir: &str) {
     let bldr = cert::CertTemplateBuilder::<EcdsaSha384Algo>::new()
         .add_basic_constraints_ext(true, 0)
         .add_key_usage_ext(usage)
-        .add_dev_sn_ext(&[0xFF; 8])
+        .add_ueid_ext(&[0xFF; 8])
         .add_dice_tcb_info_ext(
             0,
             &[
                 FwidParam {
-                    name: "TCB_INFO_FMC_HASH",
+                    name: "TCB_INFO_FMC_TCI",
                     fwid: Fwid {
                         hash_alg: asn1::oid!(/*sha384*/ 2, 16, 840, 1, 101, 3, 4, 2, 2),
                         digest: &[0xCD; 48],
                     },
                 },
                 FwidParam {
-                    name: "TCB_INFO_FMC_CONFIG",
+                    name: "TCB_INFO_OWNER_PK_HASH",
```

Comment:
The current spec says something about measuring security fuse state into FMC alias, although it's not super specific. Not sure if there are things other than FW hash and owner pk.